controllers/admin/AdminCmsController.php

```diff
@@ -135,13 +136,26 @@ public function renderForm()
                 ),
                 array(
                     'type' => 'text',
-                    'label' => $this->trans('Meta title', array(), 'Admin.Global'),
+                    'label' => $this->trans('Title', array(), 'Admin.Global'),
                     'name' => 'meta_title',
                     'id' => 'name', // for copyMeta2friendlyURL compatibility
                     'lang' => true,
                     'required' => true,
                     'class' => 'copyMeta2friendlyURL',
-                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Notifications.Info') . ' &lt;&gt;;=#{}',
+                    'hint' => array(
+                        $this->trans('Used in the h1 page tag, and as the default title tag value.', array(), 'Admin.Design.Help'),
+                        $this->trans('Invalid characters:', array(), 'Admin.Notifications.Info') . ' &lt;&gt;;=#{}',
+                    ),
+                ),
+                array(
+                    'type' => 'text',
+                    'label' => $this->trans('Meta title', array(), 'Admin.Global'),
+                    'name' => 'head_seo_title',
+                    'lang' => true,
+                    'hint' => array(
+                        $this->trans('Used to override the title tag value. If left blank the default title value is used.', array(), 'Admin.Design.Help'),
```

Comment:
I would add a comma to have it clearer: _Used to override the title tag value. If left blank, the default title value is used._

Reply:
@jolelievre